AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_MaterialAdded.py

```diff
@@ -143,56 +193,155 @@ def AtomEditorComponents_Material_AddedToEntity():
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.mesh_asset,
+            mesh_component.get_component_property_value(AtomComponentProperties.mesh('Mesh Asset')) == model.id)
+
+        # 13. Wait for Model Materials to indicate count 5 terminate early if container fails to reflect correct count
+        Report.critical_result(Tests.model_material_count, TestHelper.wait_for_condition(
+            lambda: (material_component.get_container_count(
+                AtomComponentProperties.material('Model Materials')) == 5), timeout_in_seconds=5))
+
+        # 14. Get the slot zero material from the container property Model Materials (materials defined in the fbx)
+        item = material_component.get_container_item(AtomComponentProperties.material('Model Materials'), key=0)
+        # item is an EditorMaterialComponentSlot which defines a number of method interfaces found in
+        # .\o3de\Gems\AtomLyIntegration\CommonFeatures\Code\Source\Material\EditorMaterialComponentSlot.cpp
+        label = item.GetLabel()
+        Report.result(Tests.model_material_label, label == 'lambert0')
+        # Asset path for lambert0 is 'objects/sphere_5lods_lambert0_11781189446760285338.azmaterial'; numbers may vary
+        default_asset = Asset(item.GetDefaultAssetId())
+        default_asset_path = default_asset.get_path()
+        Report.result(Tests.model_material_asset_path, default_asset_path.startswith('objects/sphere_5lods_lambert0_'))
+
+        # 15. Enable the use of LOD materials
+        material_component.set_component_property_value(AtomComponentProperties.material('Enable LOD Materials'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enable_lod_materials,
+            material_component.get_component_property_value(
+                AtomComponentProperties.material('Enable LOD Materials')) is True)
+
+        # 16. Wait for LOD Materials to indicate count 5 terminate early if container fails to reflect correct count
+        Report.critical_result(Tests.lod_material_count, TestHelper.wait_for_condition(
+            lambda: (material_component.get_container_count(
+                AtomComponentProperties.material('LOD Materials')) == 5), timeout_in_seconds=5))
+
+        # 17. Get the slot zero list from LOD Material
+        item = material_component.get_container_item(
+            AtomComponentProperties.material('LOD Materials'), key=0)  # list of EditorMaterialComponentSlot
+        active_asset = Asset(item[0].GetActiveAssetId())
+        active_asset_path = active_asset.get_path()
+        Report.result(
+            Tests.lod_material_asset_path,
+            active_asset_path.startswith('objects/sphere_5lods_lambert0_'))
+
+        # Setup a material for overrides in further testing
+        material_path = os.path.join('Materials', 'Presets', 'PBR', 'metal_gold.azmaterial')
```

Comment:
If referencing product assets I believe all of these need to be lowercase to work on Mac and Linux